kubelet/datadog_checks/kubelet/__about__.py

```diff
@@ -2,4 +2,4 @@
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
 
-__version__ = "3.4.0"
+__version__ = "3.5.0"
```

Comment:
Please remove this and the changelog. This is handled by release after the merge.

Reply:
ok